### DISCUSSION TITLE
wideLines and wedgeLines

### DISCUSSION THREAD

**Discussion by Galaxy-Man** [Bodmer/TFT_eSPI#2578] [Q&A - General] [answered]
Anyone know how to not have a radius on end of the lines?

If not can Bodmer pleas eadd this as an option to turn on/off

Thanks

**Comment by Bodmer** [comment, marked as answer]
That option is not available, it is a planned enhancement but it will not be implemented for some time due to the effort involved.

**Reply by Galaxy-Man** [reply]
Thank you prompt reply.
